tests/framework/Samplers/Restart/gold/csv/CSVdump.csv

```diff
@@ -1,26 +1,16 @@
-x1,x2,ans,ProbabilityWeight,prefix
-1.9971848083653447,2.4170219985342167,1.0,1.0,1.0
-1.93255736141758,2.7203244894557455,1.0,1.0,2.0
-1.1281244478021106,2.00011438108052,1.0,1.0,3.0
-1.9990405156274889,2.3023325678199367,1.0,1.0,4.0
-1.2360889763189686,2.1467558925847414,1.0,1.0,5.0
-1.3965807262334462,2.0923385955608307,1.0,1.0,6.0
-1.3879107403075115,2.186260211324845,1.0,1.0,7.0
-1.6697460405225275,2.3455607251603063,1.0,1.0,8.0
-1.9355390728301227,2.396767468749724,1.0,1.0,9.0
-1.8463109184164348,2.5388167320142534,1.0,1.0,10.0
-1.9971848083653447,2.4170219985342167,1.0,1.0,1
-1.9325573614175799,2.7203244894557455,1.0,1.0,2
-1.1281244478021106,2.00011438108052,1.0,1.0,3
-1.9990405156274889,2.3023325678199367,1.0,1.0,4
-1.2360889763189686,2.1467558925847414,1.0,1.0,5
-1.3965807262334462,2.0923385955608307,1.0,1.0,6
-1.3879107403075115,2.186260211324845,1.0,1.0,7
-1.6697460405225275,2.3455607251603063,1.0,1.0,8
-1.9355390728301227,2.396767468749724,1.0,1.0,9
-1.8463109184164348,2.5388167320142534,1.0,1.0,10
-1.3132735130640849,2.4191945191517461,2.0,1.0,11
-1.5245481628283271,2.6852195008390627,2.0,1.0,12
-1.4434528938595794,2.2044522490362759,2.0,1.0,13
-1.2295772200984827,2.8781174372132208,2.0,1.0,14
-1.5344139073357019,2.0273875959746976,2.0,1.0,15
+x1,x2,ans
```

Comment:
As we discussed, we should avoid regolding the tests, just do
```<what>input, output, metadata|ProbabilityWeight, .... </what>

Reply:
The data I want to have as part of this test does not include the prefix or probability weights, so I regolded.